[0m22:19:02.446833 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74dcd8684b90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74dcd8859c90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74dcd8859b50>]}


============================== 22:19:02.449368 | 82000b4e-3330-4e87-9041-ff6739a5764f ==============================
[0m22:19:02.449368 [info ] [MainThread]: Running with dbt=1.8.3
[0m22:19:02.449868 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'profiles_dir': '/home/micasidad/.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:19:09.305380 [error] [MainThread]: Encountered an error:

[0m22:19:09.315958 [error] [MainThread]: Traceback (most recent call last):
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/dbt/cli/requires.py", line 138, in wrapper
    result, success = func(*args, **kwargs)
                      ^^^^^^^^^^^^^^^^^^^^^
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/dbt/cli/requires.py", line 101, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/dbt/cli/main.py", line 470, in init
    results = task.run()
              ^^^^^^^^^^
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/dbt/task/init.py", line 325, in run
    project_name = self.get_valid_project_name()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/dbt/task/init.py", line 279, in get_valid_project_name
    name = click.prompt("Enter a name for your project (letters, digits, underscore)")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/click/termui.py", line 164, in prompt
    value = prompt_func(prompt)
            ^^^^^^^^^^^^^^^^^^^
  File "/home/micasidad/anaconda3/lib/python3.11/site-packages/click/termui.py", line 147, in prompt_func
    raise Abort() from None
click.exceptions.Abort

[0m22:19:09.318412 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 6.904774, "process_user_time": 0.938466, "process_kernel_time": 0.159826, "process_mem_max_rss": "125992", "process_in_blocks": "118272", "process_out_blocks": "16", "command_success": false}
[0m22:19:09.319141 [debug] [MainThread]: Command `dbt init` failed at 22:19:09.319021 after 6.91 seconds
[0m22:19:09.319622 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74dcd86843d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74dceb1ac890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x74dcd86de2d0>]}
[0m22:19:09.320109 [debug] [MainThread]: Flushing usage events
[0m22:39:19.113853 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x760e280355d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x760e264cfcd0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x760e264cf090>]}


============================== 22:39:19.115882 | 61300207-ab4d-4b08-a727-8bdeb8611c47 ==============================
[0m22:39:19.115882 [info ] [MainThread]: Running with dbt=1.8.3
[0m22:39:19.116472 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/micasidad/.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m22:39:33.275885 [debug] [MainThread]: Starter project path: /home/micasidad/anaconda3/lib/python3.11/site-packages/dbt/include/starter_project
[0m22:39:33.289289 [info ] [MainThread]: 
Your new dbt project "youtube_comment" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m22:39:33.289883 [info ] [MainThread]: Setting up your profile.
[0m22:42:28.222335 [info ] [MainThread]: Profile youtube_comment written to /home/micasidad/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m22:42:28.223616 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 189.14236, "process_user_time": 1.353314, "process_kernel_time": 0.196845, "process_mem_max_rss": "199352", "process_in_blocks": "129640", "process_out_blocks": "96"}
[0m22:42:28.224351 [debug] [MainThread]: Command `dbt init` succeeded at 22:42:28.224223 after 189.14 seconds
[0m22:42:28.224850 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x760e264eb390>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x760e38fb0810>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x760e38e75f90>]}
[0m22:42:28.225363 [debug] [MainThread]: Flushing usage events
[0m22:42:34.542958 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x781841997310>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x781841996f50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x781841996e90>]}


============================== 22:42:34.545183 | fde8c6e3-a490-4eb0-8167-16a450783122 ==============================
[0m22:42:34.545183 [info ] [MainThread]: Running with dbt=1.8.3
[0m22:42:34.545854 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/home/micasidad/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'invocation_command': 'dbt debug', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m22:42:34.560742 [info ] [MainThread]: dbt version: 1.8.3
[0m22:42:34.561232 [info ] [MainThread]: python version: 3.11.5
[0m22:42:34.561692 [info ] [MainThread]: python path: /home/micasidad/anaconda3/bin/python
[0m22:42:34.562082 [info ] [MainThread]: os info: Linux-6.5.0-41-generic-x86_64-with-glibc2.35
[0m22:42:35.059542 [info ] [MainThread]: Error importing adapter: No module named 'dbt.adapters.postgres'
[0m22:42:35.060679 [info ] [MainThread]: Error importing adapter: No module named 'dbt.adapters.postgres'
[0m22:42:35.063200 [info ] [MainThread]: Using profiles dir at /home/micasidad/.dbt
[0m22:42:35.063586 [info ] [MainThread]: Using profiles.yml file at /home/micasidad/.dbt/profiles.yml
[0m22:42:35.064052 [info ] [MainThread]: Using dbt_project.yml file at /home/micasidad/Desktop/yt-comment-analysis/dags/dbt/dbt_project.yml
[0m22:42:35.064487 [info ] [MainThread]: Configuration:
[0m22:42:35.064851 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m22:42:35.065212 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m22:42:35.065582 [info ] [MainThread]: Required dependencies:
[0m22:42:35.065963 [debug] [MainThread]: Executing "git --help"
[0m22:42:35.075778 [debug] [MainThread]: STDOUT: "b"usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m22:42:35.076375 [debug] [MainThread]: STDERR: "b''"
[0m22:42:35.076760 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m22:42:35.077168 [info ] [MainThread]: Connection test skipped since no profile was found
[0m22:42:35.077683 [info ] [MainThread]: [31m2 checks failed:[0m
[0m22:42:35.078118 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Credentials in profile "custom_postgres", target "dev" invalid: Runtime Error
    Could not find adapter type postgres!

Runtime Error
  Credentials in profile "postgres", target "dev" invalid: Runtime Error
    Could not find adapter type postgres!


[0m22:42:35.078544 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/micasidad/Desktop/yt-comment-analysis/dags/dbt/dbt_project.yml> not found

[0m22:42:35.079362 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.5709926, "process_user_time": 1.397229, "process_kernel_time": 0.13573, "process_mem_max_rss": "201108", "process_in_blocks": "64", "process_out_blocks": "16", "command_success": false}
[0m22:42:35.079912 [debug] [MainThread]: Command `dbt debug` failed at 22:42:35.079827 after 0.57 seconds
[0m22:42:35.080325 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7818544ac890>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7818419ffe50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7818419fef50>]}
[0m22:42:35.080799 [debug] [MainThread]: Flushing usage events
